src/wmnds/components/table/_properties.njk.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
Make sure all of these are in the same order as the example for consistency.
For example, title is above caption in the example and head is above rows.